UI/PauseScreen.cpp

```diff
@@ -321,6 +321,9 @@ void GamePauseScreen::CreateViews() {
 	continueChoice->OnClick.Handle<UIScreen>(this, &UIScreen::OnBack);
 
 	std::string gameId = g_paramSFO.GetValueString("DISC_ID");
+	if (gameId.empty() || gameId == "UCJS10041") {
```

Comment:
There are actually a lot of IDs used by homebrew, including `HBRW00000`.  Some also have no disc ID, but have a version.  I've also sometimes seen wrong game titles or screenshots reported under http://report.ppsspp.org/games, because some homebrew reused a game's ID.

I'm not sure how often it's been English homebrew, though.  Of course, there are also patched games, but that's a separate problem.

Not sure if we do or don't want reporting data for homebrew, though we do currently get some.

-[Unknown]